### DISCUSSION TITLE
Passing / accessing multiple variables into / from Send Request block

### DISCUSSION THREAD

**Discussion by namhoaingo** [postmanlabs/postman-flows#351] [Q&A] [answered]
My requests needs couple inputs from different sources. For example the following, my request needs DateTime and String. 

![image](https://user-images.githubusercontent.com/7749981/180351442-20603f93-f67c-4edd-b3f9-e51c18a331cf.png)

Within Send Request, I try to parse the variables from the current values as below 
![image](https://user-images.githubusercontent.com/7749981/180351586-f0da508b-cae8-4671-86d5-c95ac1efd412.png)

However, I can never successfully access the correct value. Looks like I can only pass a single variable into my SendRequest block. I have tried the following access path 

/variables/
/variables/0
/variables/1

Please give some advise.

**Comment by namhoaingo** [comment]
Any update on this question?

**Comment by samay-v** [comment, marked as answer]
@namhoaingo You can combine the 2 data points into one record using the `create data` block today. 
We're working on a better solution to create data and that should be available soon. When we do that, I'll update this thread 

Here's an example of it: https://www.postman.com/postman/workspace/postman-answers/flow/62e37cac5f71fd2a1ec21013

![image](https://user-images.githubusercontent.com/28357106/181696822-55cd4f3c-7a7f-49fb-b603-929e87b32f21.png)

**Reply by namhoaingo** [reply]
Thank @samay-v
